torch_xla/experimental/xla_sharding.py

```diff
@@ -74,4 +74,7 @@ def mark_sharding(t: Union[torch.Tensor,
 
 def clear_sharding(t: Union[torch.Tensor, XLAShardedTensor]) -> torch.Tensor:
   """Clear sharding annotation from the input tensor and return a `cpu` casted tensor."""
-  return NotImplemented
+  torch_xla._XLAC._xla_clear_sharding(t)
+  if isinstance(t, XLAShardedTensor):
```

Comment:
in what case we will call `clear_sharding` on a non-XLAShardedTensor?

Reply:
In the example above, one may call `xs.claer_sharding` on `xt1` just like `mark_sharding`.

Reply:
ok sg.